client/src/document/lazy-bcd-table.tsx

```diff
@@ -68,6 +74,15 @@ function LazyBrowserCompatibilityTableInner({ dataURL }: { dataURL: string }) {
   if (error) {
     return <p>Error loading BCD data</p>;
   }
+  if (!data.data) {
+    return (
+      <p>
+        Error loading BCD table because there's no known data for{" "}
+        <code>{query}</code>
+      </p>
```

Comment:
Cribbing a bit from the current kumascript macro text. How about something like this?

```suggestion
      <p>
        No compatibility data found for <code>{query}</code>. Check the spelling or contribute data to <a href="https://github.com/mdn/browser-compat-data">mdn/browser-compat-data</a>.
      </p>
```